lib/embedded_ansible.rb

```diff
@@ -50,6 +51,15 @@ def self.configure
   def self.start
     configure_secret_key
     run_setup_script(START_EXCLUDE_TAGS)
+
+    5.times do
+      return if alive?
+
+      _log.info("Waiting for EmbeddedAnsible to respond")
+      sleep WAIT_FOR_ANSIBLE_SLEEP
+    end
```

Comment:
I like this better.  Now the caller can be guaranteed to be alive or get an exception. 👍